### DISCUSSION TITLE
Encounter database and Terastalised Pokemon Encounters

### DISCUSSION THREAD

**Discussion by k20stitch** [kwsch/PKHeX#3853] [Q&A]
Are all of the Terastalised Pokemon encounters supposed to be in the encounter database? There's an Umbreon found at level 50 at Alfornado Cavern. The only ones in the encounter database at this location are level 37-44.

**Comment by Lusamine** [comment]
Yes, Tera Pokemon are Fixed Symbol Encounters (categorized under "Static"). Since you provided a level range, you are looking at the wrong encounter type, Encounter Slots. This is not an error with the program.
![image](https://user-images.githubusercontent.com/30205550/229329449-e48b1a0e-0819-4ad2-bb6d-ff8ad064df6f.png)

**Reply by k20stitch** [reply]
The one caught at Alfornada cavern has Alfornada Cavern listed as the met location but otherwise the one above looks exactly like the one I caught just that the met location displayed in game is different.

**Reply by Lusamine** [reply]
It is the same Umbreon. Encounters can wander across boundaries, so multiple met locations are valid. The Encounter Database shows one that is valid. If you don't believe me, select it and change the met location.
